collection_files/motogp_2024.yml

```diff
@@ -80,3 +80,12 @@ collections:
         episode_year: 2024
     url_poster: https://raw.githubusercontent.com/s0len/meta-manager-config/main/posters/motogp_2024/R09.jpg
     sort_title: MotoGP 09 
+  Round 10 Grand Prix Deutschland:
+    builder_level: episode
+    smart_filter:
+      all:
+        title: MotoGP
+        episode_title: Grand Prix British
```

Comment:
Change this to only contain British (if that's what it's called on tvdb? I think it might be Britain).
Just a note on this line, it is a search for all files which contains that string